frontend/src/components/ColumnChartPreview.tsx

```diff
@@ -50,8 +52,15 @@ const ColumnChartPreview = (props: Props) => {
     }
   };
 
+  const widthPercent =
+    (UtilsService.getLargestHeader(columns, data) *
+      (data ? data.length : 0) *
+      8 *
+      100) /
+    (props.isPreview ? 480 : 960);
```

Comment:
Similar comment. `480`, `960`, `8 * 100`, `0`. What do these numbers mean? Are we going to remember in the future? What about making them constants on the top of the file to give them descriptive names at least? Would that help?

Reply:
Same